FirebaseAnalytics.podspec.json

```diff
@@ -22,7 +22,7 @@
         "z"
     ],
     "license": {
-        "text": "Copyright 2021 Google",
+        "text": "Copyright 2022 Google",
```

Comment:
Technically Copyrights only need to be updated when the file is created, but we have been updating them for podspecs - so either way.

Reply:
Interesting, thanks for the info.